packages/sdk/src/types/runtime/assets/assetManager.ts

```diff
@@ -85,6 +86,9 @@ export class AssetManager {
         } as LoadAssets;
 
         const response = await this.sendLoadAssetsPayload(payload);
+        if (response.failureMessage) {
+            throw new Error(response.failureMessage);
+        }
 

```

Comment:
This needs to be converted to a rejected promise in the calling method.

Reply:
It's in an async method, it's already a promise rigged to reject on errors, right?

Reply:
Oh yes! Missed that detail.

---
In reply to: [251176960](https://github.com/Microsoft/mixed-reality-extension-sdk/pull/143#discussion_r251176960) [](ancestors = 251176960)